src/pages/Upload/Upload.js

```diff
@@ -127,43 +146,45 @@ const Upload = () => {
         </SelectBox>
       </ContentBox>
       <UploadBox>
-        <UploadImg htmlFor="upload">
-          <UploadInput type="file" id="upload" />
-          <CameraIcon src="/images/camera.png" alt="cameraIcon" />
-          <UploadImgTitle>
-            {clickPhotoOrVideo ? '사진' : '동영상'} 올리기
-          </UploadImgTitle>
-          <MaxTenImgs>
-            {clickPhotoOrVideo
-              ? '(* 최대 10장까지)'
-              : '(* 5GB 미만, 3초~60초 길이의 \n세로 영상을 권장합니다.)'}
-          </MaxTenImgs>
-        </UploadImg>
-        <UploadText>
-          <SelectBox marginRight="0">
-            <Select
-              onChange={e => setSelectedSpace(e.target.value)}
-              isSelected={!!selectedSpace}
-            >
-              <Option value="none" hidden>
-                공간 (필수)
-              </Option>
-              {SPACE.map(({ id, text }) => (
-                <Option key={id} value={id}>
-                  {text}
-                </Option>
-              ))}
-            </Select>
-            <DownArrowImg src="/images/downArrow.png" alt="downArrowImg" />
-          </SelectBox>
-          <UploadTextArea placeholder="사진에 대해 설명해주세요." />
-          <TagsInput
-            value={hagTags}
-            onChange={setHashTags}
-            name="hagtag"
-            placeHolder="키워드"
+        {uploadedImgs.length === 0 ? (
+          <UploadLists
+            handleUploadImg={handleUploadImg}
+            clickPhotoOrVideo={clickPhotoOrVideo}
+            space={SPACE}
+            uploadedImgs={uploadedImgs}
           />
-        </UploadText>
+        ) : (
+          uploadedImgs.map((image, index) => (
+            <UploadLists
+              key={index}
+              image={image}
+              handleUploadImg={handleUploadImg}
+              clickPhotoOrVideo={clickPhotoOrVideo}
+              space={SPACE}
+              uploadedImgs={uploadedImgs}
+            />
+          ))
+        )}
+        <UploadPreview>
+          {uploadedImgs.length === 0
+            ? null
+            : uploadedImgs.map((item, index) => (
+                <PreviewImgs key={index} draggable>
+                  <PreviewImg src={item} />
+                </PreviewImgs>
+              ))}
```

Comment:
length가 0이라면 map 메서드의 출력 결과가 없기 때문에, 조건에 따라 분기처리하시지 않아도 됩니다!
```suggestion
          {uploadedImgs.map((item, index) => (
                <PreviewImgs key={index} draggable>
                  <PreviewImg src={item} />
                </PreviewImgs>
              ))}
```